nipype/interfaces/ants/registration.py

```diff
@@ -204,11 +204,7 @@ def _transformation_constructor(self):
         return "".join(retval)
 
     def _regularization_constructor(self):
-        return "--regularization {}[{},{}]".format(
-            self.inputs.regularization,
-            self.inputs.regularization_gradient_field_sigma,
-            self.inputs.regularization_deformation_field_sigma,
-        )
+        return f"--regularization {self.inputs.regularization}[{self.inputs.regularization_gradient_field_sigma},{self.inputs.regularization_deformation_field_sigma}]"
```

Comment:
This doesn't strike me as more readable. Could do something like:

```suggestion
        gfsigma = self.inputs.regularization_gradient_field_sigma
        dfsigma = self.inputs.regularization_deformation_field_sigma
        return f"--regularization {self.inputs.regularization}[{gfsigma},{dfsigma}]"
```

Alternately, we could use format strings sensibly using `input_spec` fields and then `trait_get()` to populate them:

```suggestion
        fmt = "--regularization {regularization}[{regularization_gradient_field_sigma},{regularization_deformation_field_sigma}]"
        return fmt.format(**self.inputs.trait_get())
```

I would do this throughout. I don't think f-strings are improving things here, even if they might be very slightly more efficient.

Reply:
I agree. I'm not very happy with these changes. I can't decide whether I should keep UP0032 altogether and discard these changes, or apply the changes that make sense and silence the rest with `noqa`. What would you do?

Reply:
We could do a per-file ignore, if it's only a couple files that get really ugly. Then we can revisit them separately or never. `%` and `str.format` aren't going away, so there's no time bomb if we end up just saying "ANTs is too special for UP032".

If it's broad, we could set the ignore to everything in "nipype/interfaces".